.github/workflows/lint.yml

```diff
@@ -1,15 +1,27 @@
-name: PHPStan
+name: Lint
 
 on:
   pull_request:
   push:
     branches:
-      - main
+      - 'main'
       - '5.*'
 
 jobs:
-  prestashop:
-    name: ${{ matrix.ps-version }}
+  php-cs-fixer:
+    name: PHP CS Fixer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v6
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.5'
```

Comment:
I'm wondering if using PHP8.5 for PHP CS Fixer means we could apply some codestyle not compatible with PHP8.1 or PHP7 and break something for one of our target builds (PS 1.7, PS 8, PS 9)

I just ask, maybe I am paranoid

Reply:
It should not, PHP-CS-Fixer acts from a set of rules, if those rules do not contain PHP 8.5 code, it won't apply incompatible code style.